cipd_packages/ruby/README

```diff
@@ -0,0 +1,24 @@
+# Ruby at Flutter
+
+The scripts in this folder are used to build and package
+ruby as a CIPD package. This package is intended for
+flutter/flutter repository testing.
+
+The scripts inside ruby_ship are based on the scripts
+from https://github.com/stephan-nordnes-eriksen/ruby_ship
+with changes to adapt them to the flutter use cases.
+
+
+# Updating to new ruby versions.
+
+Update `ruby_version.txt` with the major version
```

Comment:
If this is existing, we can carry it over, but if not, maybe rename the file to be something different, since it doesn't only contain the version number, but also filename as well.

ruby_metadata.txt or info?

Reply:
done!